.github/workflows/release.yml

```diff
@@ -1,24 +1,34 @@
+# This workflow should only be used for version releases for cadenza.js versions bundled with
+# Cadenza 10.2 and later. Starting with 10.2 we do not use Semantic versioning in the classic way any longer.
+# For earlier versions use the "Legacy Release" workflow.
+
 name: Release
 
 on:
   workflow_dispatch:
     inputs:
-      release-type:
-        type: choice
-        description: 'Release type'
-        required: true
-        options:
-          - patch
-          - minor
-          - major
-        default: 'patch'
+      cadenza-version:
+        type: string
+        description: |
+          Cadenza Version
+          (When given in format x.x, a major x.x.0 version is released. Otherwise, the patch version number of the
+          current version is increased)'
```

Comment:
Good to have such a hint here (y) I think we should stress more that this param is optional. Proposal:

> Required only to create a new <version>.0 release (e.g. 10.2.0). Otherwise the current version is incremented.

Reply:
Done.